WalletWasabi.Fluent/Views/CoinControl/Core/Cells/LabelsCellView.axaml

```diff
@@ -0,0 +1,39 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:controls="clr-namespace:WalletWasabi.Fluent.Controls"
+             xmlns:cells="clr-namespace:WalletWasabi.Fluent.ViewModels.CoinControl.Core.Cells"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:Class="WalletWasabi.Fluent.Views.CoinControl.Core.Cells.LabelsCellView"
+             x:DataType="cells:LabelsCellViewModel"
+             x:CompileBindings="True">
+
+  <UserControl.Resources>
+    <SolidColorBrush x:Key="PrivacyLevelNoneBrush" Color="{DynamicResource TextForegroundColor}" />
```

Comment:
This should be moved to same place as PrivacyLevelMediumBrush and PrivacyLevelStrongBrush  ?

Reply:
No, in this case we are overriding that resource because the base one isn't correct in this view.